code/modules/surgery/surgery_step.dm

```diff
@@ -174,10 +174,7 @@
 			chems += chemname
 	return english_list(chems, and_text = require_all_chems ? " and " : " or ")
 
-//Replaces visible_message during operations so only people looking over the surgeon can tell what they're doing, allowing for shenanigans.
+//Replaces visible_message during operations so only people looking over the surgeon can see them.
 /datum/surgery_step/proc/display_results(mob/user, mob/living/carbon/target, self_message, detailed_message, vague_message, target_detailed = FALSE)

```

Comment:
Remove target_detailed, this deprecates it

Reply:
Good point. Now to remove it from 200 calls aaaaaaaaaaaa

Reply:
Wait no

Reply:
I fucked it up